### PR TITLE
[Merged by Bors] - feat: provide http sink documentation

### PR DESCRIPTION
**Pull request by EstebanBorai**
## Demo

image

### CODE REVIEW

**Review by ajhunyady — APPROVED** [review]
LGTM

**Comment by ajhunyady** [comment]
thought this looks like it's still in Draft.

**Comment by EstebanBorai** [comment]
Hi @ajhunyady! I just updated missing bits from docs with latest updates!

File: embeds/connectors/outbound/http.md
Comment (morenol) [inline embeds/connectors/outbound/http.md:1]: We use to keep this file on sync with connector readme in the repo. Could you update this first there?

**Review by morenol — CHANGES_REQUESTED** [review]

File: embeds/connectors/outbound/http.md
Comment (EstebanBorai) [reply embeds/connectors/outbound/http.md:1]: Sure thing!

**Comment by EstebanBorai** [comment]
Copying manually from https://github.com/infinyon/http-sink-connector/pull/23

**Review by morenol — APPROVED** [review]

**Review by ajhunyady — APPROVED** [review]
Looks good.

**Pull request closed without merging**